leshan-core/src/main/java/org/eclipse/leshan/json/JsonArrayEntry.java

```diff
@@ -155,7 +157,7 @@ public boolean equals(Object obj) {
                 return false;
         } else if (!stringValue.equals(other.stringValue))
             return false;
-        if (time != other.time)
```

Comment:
There is an mistake here, this method was modified manually. We should regenerated with eclipse : 
```java
if (time == null) {
    if (other.time != null)
        return false;
} else if (!time.equals(other.time))
    return false;
```